cluster-autoscaler/Dockerfile

```diff
@@ -11,7 +11,7 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-ARG BASEIMAGE=gcr.io/distroless/static:latest
+ARG BASEIMAGE=gcr.io/distroless/base:latest
```

Comment:
sinice?

Reply:
It was initially working for me with the distroless/static base image. After some point I assume something changed in the latest version of that image and my home baked cluster-autoscaler containers would fail to start with the following message:
```
standard_init_linux.go:207: exec user process caused "no such file or directory"
```
After switching to the distroless/base image things were back in order. I didn't investigate deeper.

Reply:
This is a global change that may impact other providers/environment. Please do it in a separate PR if really needed.

Reply:
@d-mo Is the change of base image related to this PR. Or master branch stopped working with `gcr.io/distroless/static:latest`?

Reply:
I reverted the base image change since it was not related to this PR. I will investigate what the problem was and under which circumstances it occurs and I'll open a new issue/PR if it turns out to be necessary.